extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/discovery/HueLightDiscoveryService.java

```diff
@@ -103,22 +104,25 @@ private void onLightAddedInternal(FullLight light) {
         ThingUID thingUID = getThingUID(light);
         ThingTypeUID thingTypeUID = getThingTypeUID(light);
 
+        String modelId = light.getModelID().replaceAll(HueLightHandler.NORMALIZE_ID_REGEX, "_");
+
         if (thingUID != null && thingTypeUID != null) {
             ThingUID bridgeUID = hueBridgeHandler.getThing().getUID();
             Map<String, Object> properties = new HashMap<>(1);
             properties.put(LIGHT_ID, light.getId());
+            properties.put(MODEL_ID, modelId);
 
             /*
              * TODO retrieve the light´s unique id (available since Hue bridge versions > 1.3) and set the mac address
-             * as discovery result representationÏ. For this purpose the jue library has to be modified.
+             * as discovery result representation. For this purpose the jue library has to be modified.
              */
 
             DiscoveryResult discoveryResult = DiscoveryResultBuilder.create(thingUID).withThingType(thingTypeUID)
                     .withProperties(properties).withBridge(bridgeUID).withLabel(light.getName()).build();
 
             thingDiscovered(discoveryResult);
         } else {
-            logger.debug("discovered unsupported light of type '{}' with id {}", light.getModelID(), light.getId());
+            logger.debug("discovered unsupported light of type '{}' with id {}", modelId, light.getId());
```

Comment:
Ok, to be more precise, you should log the input from which the thing type id is derived, namely light.getType(), i.e. the ZigBee type. Because if this is of a value that we do not have in our "supported map", we will end up in this branch, right?